aircraft.py

```diff
@@ -132,7 +136,7 @@ def __init__(
         self.orientation = 1
         self.flipstart = 0.0
 
-        # Dependent variables (oa Numpy containers)
+        # Dependent variables (e.g., Numpy containers)
         self.pos_virtual = self.pos_real * \
                            settings.PLANE_POS_SCALE % \
                            self.window_dimensions
```

Comment:
not very pep of you

Reply:
![afbeelding](https://github.com/Finnyy012/TUPOLEVGAMING/assets/94832237/99210a7e-eb2d-4c3a-931a-03fa22821219)